src/App.js

```diff
@@ -1,30 +1,31 @@
 import "./App.css";
 
-import {LandingPage} from "./pages/Landingpage/Landing";
-import {  Routes, Route,} from "react-router-dom";
-import {Explore} from "./pages/Landingpage/ExplorePage/Explore";
-import {Video} from './components/video/Video';
-import {LikePage} from './pages/LikePage/LikePage';
-import {WatchLaterPage} from './pages/watchLater/Watchlater'
-
-
+import { LandingPage } from "./pages/Landingpage/Landing";
+import { Routes, Route } from "react-router-dom";
+import { Explore } from "./pages/Landingpage/ExplorePage/Explore";
+import { Video } from "./components/video/Video";
+import { LikePage } from "./pages/LikePage/LikePage";
+import { WatchLaterPage } from "./pages/watchLater/Watchlater";
+import { PlayListCraetePage } from "./components/playlistmodalcard/createPlaylistPage";
+import { InnerPlayListPage } from "./pages/innerplaylistpage/innerPlayList";
+import { HistoryPage } from "./pages/HistoryPage/HistoryPage";
```

Comment:
Please use barrels instead of importing all the files here. Create a common index file.